runtime/processor.go

```diff
@@ -134,6 +134,17 @@ func (r *Processor) Listen() {
 				planId = primitive.NilObjectID
 			}
 
+			var expiredStatus string
+
+			// Check if the record has expired
+			if time.Now().After(msg.TTL) {
+				fmt.Println("The record has expired.")
+				expiredStatus = "expired"
+			} else {
+				fmt.Println("The record is still valid.")
+				expiredStatus = "valid"
+			}
+
```

Comment:
We don't need an expired status here. Having a status would mean we will constantly be updating records when they expire. 
When searching results, instead of a status, we will do something like: 
```
active := "select * from results where expires >= now()"
expired := "select * from results where expires < now()"
```

In the UI, we will do a similar thing:
```
if (result.expires >= now()){
   showActiveLabel();
} else {
  showExpiredLabel();
}
```

When exporting an OSCAL document we will always only filter for active results:
```
select * from results where expires >= now();
```

Status would not mean much to us, and might open holes if we miss updating a record status.